website/components/PropTable.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import { css } from 'emotion';
+import { Table, TableHeader, Row, Cell } from '@leafygreen-ui/table';
+import { Subtitle, InlineCode } from '@leafygreen-ui/typography/dist';
+import PropDefinition from 'components/PropDefinition';
+import TypographyPropTable from 'components/TypographyPropTable';
+
+const subtitleBottomMargin = css`
+  margin-bottom: 24px;
+`;
+
+const tableBottomMargin = css`
+  margin-bottom: 56px;
+`;
+
+/**
+ * Syntactic units in unist syntax trees are called nodes.
+ * Interface dervied from Unified types
+ */
+export interface Node {
+  /**
+   * The variant of a node.
+   */
+  type: string;
+
+  /**
+   * Information from the ecosystem.
+   */
+  data?: {
+    [key: string]: unknown;
+  };
+
+  /**
+   * Location of a node in a source document.
+   * Must not be present if a node is generated.
+   */
+  position?: Position;
+
+  children?: Array<any>;
+
+  [key: string]: unknown;
+}
+
+const map = {
+  [0]: 'prop',
+  [1]: 'type',
+  [2]: 'description',
+  [3]: 'default',
+} as const;
```

Comment:
could use a more descriptive variable name. also, why not an array?

Reply:
👍